suite/tests/client-interface/drreg-test.dll.c

```diff
@@ -149,6 +150,13 @@ static dr_emit_flags_t
 event_app_analysis(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
                    bool translating, OUT void *user_data)
 {
+    for (instr_t* instr = instrlist_first(bb); instr != NULL; instr = instr_get_next(instr)) {
+        reg_id_t r;
+        drreg_status_t res;
+        bool dead;
+        CHECK(drreg_are_aflags_dead(drcontext, instr, &dead)==DRREG_SUCCESS, "drreg_are_aflags_dead should always work");
+        CHECK(drx_aflags_are_dead(instr)==dead, "aflag liveness estimation of drx and drreg should always consist");
```

Comment:
Are there any existing tests that hit this assert? If yes, then we can probably remove the new `test15` in drreg-test. If no, then please add a comment here that this is relevant to test15.

Reply:
I have tested without newly added test15 and the existing test can still hit this assertion when the fix is not included. I will delete the redundant test15 in the next commit.

Reply:
The format is fixed using the latest `clang-format` tool.